components/Smiley.tsx

```diff
@@ -19,12 +19,25 @@ const EMOJI_MAP: EmojiMap = {
 }
 
 const Smiley = ({ percentage }: Props) => {
+  const emojiKeys = Object.keys(EMOJI_MAP).map(Number)
+  const [min, max] = [Math.min(...emojiKeys), Math.max(...emojiKeys)]
   const matchingEmoji = Object.keys(EMOJI_MAP).find(
     (key) => parseInt(key) >= percentage * 100
   )
   if (!matchingEmoji) return null
 
-  return <span className={styles.large}>{EMOJI_MAP[matchingEmoji]}</span>
+  return (
+    <span
+      aria-valuemax={max / 10}
+      aria-valuemin={min / 10}
+      aria-valuenow={parseInt(matchingEmoji) / 10}
```

Comment:
Are we sure these aria tags are allowed on a `span`? Should this maybe be a `progress` or similar?

Reply:
Well probably not a `progress` since that can't have children...

Reply:
what's a `progress`?